tsconfig.json

```diff
@@ -0,0 +1,28 @@
+{
+  "extends": "./tsconfig.extend.json",
+  "compilerOptions": {
+    "baseUrl": ".",
+    "target": "es5",
```

Comment:
why are we targeting ES5? ES6 is like 5-6 years old at this point and is supported by basically everything. is there a specific reason to target ES5?

Reply:
this was the default value from Create React App. I just forgot to change it! Woops